test/square.test.js

```diff
@@ -0,0 +1,13 @@
+const { square } = require('../lib');
+
+test('0 squared to equal 0', () => {
+  expect(square(0)).toBe(0);
+});
+
+test('1 squared to equal 1', () => {
+  expect(square(1)).toBe(1);
+});
+
+test('3 squared to equal 9', () => {
+  expect(square(1)).toBe(1);
```

Comment:
@hugoduraes I think this is meant to be `(square(9)).toBe(3)`! Fix that and it's in!

Reply:
Damn you copy paste! :(

Reply:
haha it happens!

Reply:
Fixed now!